hoot-core/src/main/cpp/hoot/core/elements/ElementIterator.h

```diff
@@ -22,11 +22,11 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015, 2017 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2015, 2017, 2018 DigitalGlobe (http://www.digitalglobe.com/)
  */
 
-#ifndef ELEMENTPROVIDER_H
-#define ELEMENTPROVIDER_H
+#ifndef ELEMENTITERATOR_H
+#define ELEMENTITERATOR_H
```

Comment:
At some point the file was either renamed or it was copied from another and the `#define` wasn't updated.  Fixed it.